src/mimefactory.rs

```diff
@@ -637,6 +637,11 @@ impl<'a> MimeFactory<'a> {
                     "Content-Type".to_string(),
                     "multipart/report; report-type=multi-device-sync".to_string(),
                 ))
+            } else if self.msg.param.get_cmd() == SystemMessage::W30StatusUpdate {
+                PartBuilder::new().header((
+                    "Content-Type".to_string(),
+                    "multipart/report; report-type=status-update".to_string(),
```

Comment:
well, `multipart/report`, [RFC 6522](https://datatracker.ietf.org/doc/html/rfc6522), is the "type for electronic mail reports of any kind", i think that fits somehow. it is used on large scale for mdn and we are already using it internally for multi-device-sync as of https://github.com/deltachat/deltachat-core-rust/pull/2669

it would be a huge step forward, if other mua would stop displaying these messages as "normal messages" :)